Documentation/Input/InputState.md

```diff
@@ -0,0 +1,54 @@
+# Accessing Input State in MRTK
+It's possible to directly query the state of all inputs in MRTK by iterating over the controllers attached to the input sources. MRTK also provides convenience methods for accessing the position and rotation of the eyes, hands, head, and motion controller.
+
+See the InputDataExample scene for an example of querying input both via iterating over controllers, and by using the [`InputUtils`](cref:Microsoft.MixedReality.Toolkit.Input.InputUtils) class.
+
+## Example: Access position, rotation of head, hands, eyes in MRTK
+MRTK's [`InputUtils`](cref:Microsoft.MixedReality.Toolkit.Input.InputUtils) class provies convenience methods for accessing the hand ray, head ray, eye gaze ray, and motion controller rays.
+
+```csharp
+// Get the head ray
+var headRay = InputUtils.GetHeadGazeRay();
+
+// Get the right hand ray
+Ray rightHandRay;
+if(InputUtils.TryGetHandRay(Handedness.right, rightHandRay))
+{
+    // Right hand ray is available
+}
+else
+{
+    // Right hand is not available
```

Comment:
comment is wrong

Reply:
Are you sure?

Reply:
Oh lol I read too fast and saw it as duplicated